### DISCUSSION TITLE
Bt adapter

### DISCUSSION THREAD

**Discussion by maxplayz2746** [pybricks/support#804] [Q&A] [answered]
how would I fix the problem with py bricks saying "No Bluetooth adapter could be found. Please connect or enable a Bluetooth Low Energy adapter and restart the browser." with a built in Bluetooth adapter

**Comment by dlech** [comment, marked as answer]
Are you using Pybricks Code, Pybricks Beta, pybricksdev, something else? What operating system are you using? What web browser are you using? Is Bluetooth turned off either in software or with a hardware switch?

**Reply by maxplayz2746** [reply]
py bricks code windows 11 opera gx it is on and its on in software

**Reply by dlech** [reply]
If I am remembering right, Bluetooth may be disabled in the Opera browser. Can you try a different browser that support Web Bluetooth like Edge or Chrome?

**Comment by maxplayz2746** [comment]
![Screenshot 2022-11-15 180129](https://user-images.githubusercontent.com/84992927/202063797-c21d3a45-9eb6-4e7f-a7bf-bfc3d24dce34.png)
that is what's in my Bluetooth section
